docs/snakefiles/reporting.rst

```diff
@@ -246,36 +254,77 @@ Further, using ``url_args()`` you can add URL arguments and using ``anchor()`` y
             file=f,
         )
 
+.. _snakefiles-rendering_reports:
+
 Rendering reports
 -----------------
 
-To create the report simply run
+All the metadata contained in the report (e.g. runtime statistics) is automatically collected during the rendering of the report.
```

Comment:
"are automatically"? As metadata are plural?

Reply:
I went back and forth on this. Grammatically correct is the plural, but singular is also acceptable. So let's maybe go with grammatically correct:
```suggestion
All the metadata contained in the report (e.g. runtime statistics) are automatically collected during the rendering of the report.
```